physics/cu_c3_deep.F90

```diff
@@ -2078,9 +2078,9 @@ subroutine cu_c3_deep_run(        &
 
 !> - Call rain_evap_below_cloudbase() to calculate evaporation below cloud base
 
-      call rain_evap_below_cloudbase(itf,ktf,its,ite,                    &
-           kts,kte,ierr,kbcon,xmb,psur,xland,qo_cup,                     &
-           po_cup,qes_cup,pwavo,edto,pwevo,pre,outt,outq)      !,outbuoy)
+!      call rain_evap_below_cloudbase(itf,ktf,its,ite,                    &
+!           kts,kte,ierr,kbcon,xmb,psur,xland,qo_cup,                     &
+!           po_cup,qes_cup,pwavo,edto,pwevo,pre,outt,outq)      !,outbuoy)
 
```

Comment:
I suggest we keep this just in case we need to tune with subcloud evaporation in the future.

Reply:
We've been recommending that folks remove code rather than comment it out because folks often forget why they left commented code in and it makes the code harder to read in the long run. Git saves the history, so one can always go back to a commit before this PR and put these lines back in.

Reply:
Ok, I will remove these commented lines.